@xen-orchestra/async-map/package.json

```diff
@@ -46,7 +46,6 @@
     "dev": "cross-env NODE_ENV=development babel --watch --source-maps --out-dir=dist/ src/",
     "prebuild": "yarn run clean",
     "predev": "yarn run prebuild",
-    "prepare": "yarn run build",
```

Comment:
Something went wrong when you merged/rebase your branch.

Reply:
Corrected (after a long battle with Git/Gitkraken/IntelliJ/GitHub)